subprojects/core/core.gradle.kts

```diff
@@ -100,6 +100,9 @@ dependencies {
     testFixturesApi(project(":execution")) {
         because("test fixtures expose OutputChangeListener")
     }
+    testFixturesApi(project(":native")) {
+        because("test fixtures expose FileSystem")
+    }
```

Comment:
Why is this needed now?

Reply:
Without this I was getting, I suspect because of AST transformations or something like that.

```
org.gradle.api.tasks.TaskExecutionException: Execution failed for task ':buildCachePackaging:compileTestGroovy'.
Caused by: java.lang.NoClassDefFoundError: org.gradle.internal.nativeintegration.filesystem.FileSystem
```

This declaration was missing in core because of `org.gradle.api.internal.file.TestFiles#fileSystem()` which returns a `FileSystem`

Reply:
Got it, thanks.